src/ResourceManager/AzureBackup/Commands.AzureBackup.Test/App.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This app.config is required to get input for variable that is used for initialization of client.